core/liblwm2m.h

```diff
@@ -643,6 +663,7 @@ typedef struct
     uint16_t                nextMID;
     lwm2m_transaction_t *   transactionList;
     void *                  userData;
+    lwm2m_block1_data_t *   block1DataList;
```

Comment:
I will add the free on `lwm2m_close`.
I will see if I can move block1_data structure in `server` instance of `lwm2m_context`.